lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -789,6 +795,50 @@ def test_render_html_view_with_preview_mode_when_user_already_has_cert(self):
         self.assertIn('Signatory_Title 0', response.content)
 
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_html_view_certificate_availability_date_for_instructor_paced_courses(self):
```

Comment:
this is a good use case for `ddt.data`, so you could cover two test conditions with just this one test method.  You can do that as follows:
- Add a decorator `@ddt.data(True, False)` to this function.
- Call the function something more generic: `test_html_view_certificate_available_date_paced_courses`
- Add a parameter to the method called `is_self_paced`
- Make the first line of the method body by `self.course.self_paced = is_self_paced`
- remove the function directly beneath this one.